src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -159,11 +159,8 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 			}
 
 			var sw = Stopwatch.StartNew();
-#if DEBUG
-			if (System.Diagnostics.Debugger.IsAttached)
```

Comment:
Is this failing in debug somehow? I don't mind the change, though.

Reply:
Not this instance, but something was failing because of a `Debugger.IsAttached` condition somewhere, so I removed all these checks everywhere. I've been bitten by those before too and it always wastes a lot of time because you'd never guess that attaching the debugger would change the behaviour.